src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIO.java

```diff
@@ -48,7 +52,4 @@ public default void setLeftVelocitySetpoint(double velocityRadiansPerSecond) {}
 
   /** Sets motor closed loop velocity setpoint for right flywheel. */
   public default void setRightVelocitySetpoint(double velocityRadiansPerSecond) {}
```

Comment:
In a motion profile, the setpoint is set by the constraints of the profile, we have to set the overall goal of the profile.
```suggestion
  public default void setRightVelocityGoal(double velocityRadiansPerSecond) {}
```